notebook/static/notebook/js/maintoolbar.js

```diff
@@ -131,5 +132,17 @@ define([
 
     };
 
+    MainToolBar.prototype._pseudo_actions.change_kernel = function () {
+        var multiselectKernel = $('<option/>').attr('value','multiselect').attr('disabled','').text('-');
+        var kernelSel = $('<select/>')
+            .attr('id','change_kernel_type')
+            .attr('aria-label', i18n.msg._('combobox, change kernel'))
+            .attr('role', 'combobox')
+            .addClass('form-control select-xs')
+            .append(multiselectKernel);
+        this.notebook.keyboard_manager.register_events(kernelSel);
+        return kernelSel
```

Comment:
one liner comment on what exactly this piece of code is doing!

Reply:
Added comment explaining its function.